crates/glaredb/src/bin/main.rs

```diff
@@ -54,8 +54,12 @@ fn main() -> Result<()> {
     // Disable logging when running locally since it'll clobber the repl
     // _unless_ the user specified a logging related option.
     match (&command, cli.log_mode, cli.verbose) {
-        (Commands::Local { .. }, None, 0) => (),
-        _ => logutil::init(cli.verbose, cli.log_mode.unwrap_or_default().into()),
+        (Commands::Local(args), None, 0) => {
+            if let Some(file) = &args.log_file {
+                logutil::init(1, LoggingMode::Full.into(), Some(file))
+            }
+        }
+        _ => logutil::init(cli.verbose, cli.log_mode.unwrap_or_default().into(), None),
```

Comment:
We should be log to file with whatever the log mode or verbosity is passed using the command line. This only does it for 0 verbosity and no log mode.